### PR TITLE
feat: Add model migrate

### PR DESCRIPTION
**Pull request by adevjoe**
使用 [migrate](github.com/golang-migrate/migrate) 框架实现权限模型 migrate。

### CODE REVIEW

**Comment by tencentblueking-adm** [comment]
[CLA assistant check](https://bk-cla.bktencent.com:443/TencentBlueKing/iam-go-sdk?pullRequest=32) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://bk-cla.bktencent.com:443/TencentBlueKing/iam-go-sdk?pullRequest=32) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://bk-cla.bktencent.com:443/check/TencentBlueKing/iam-go-sdk?pullRequest=32) it.</sub>

File: util/array.go [hunk 1]
```diff
@@ -25,3 +25,16 @@ func Int64ArrayToString(input []int64, sep string) string {
...
+// The function takes a slice of elements and a target value. It returns a boolean
+// value indicating whether the target value is present in the slice or not.
+func Contains[T comparable](elems []T, v T) bool {
+	for _, s := range elems {
+		if v == s {
+			return true
+		}
+	}
+	return false
+}
```
Comment (wklken) [inline util/array.go:40]: NIT: add unittest

File: iammigrate/migrate_test.go [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+ * specific language governing permissions and limitations under the License.
+ */
+
+package iammigrate
+
+import (
+	"bytes"
+	"testing"
+)
+
```
Comment (wklken) [inline iammigrate/migrate_test.go:18]: NIT: use ginkgo + testify (BDD style)

**Comment by adevjoe** [comment]
@wklken PTAL

**Review by wklken — APPROVED** [review]
LGTM

**Comment by adevjoe** [comment]
@wklken 看下能合下，打个 tag 吗

**Comment by wklken** [comment]
> 

@zhu327

**Review by zhu327 — APPROVED** [review]

**Pull request merged**